keycloak.advisories.yaml

```diff
@@ -116,13 +124,37 @@ advisories:
           type: component-vulnerability-mismatch
           note: CVE is being considered by the community a false positive. See https://github.com/anchore/grype/issues/1068
 
+  - id: CVE-2022-45935
+    events:
+      - timestamp: 2023-10-05T13:50:00Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: CVE is being considered by the community a false positive. See https://github.com/anchore/grype/issues/1138
+
+  - id: CVE-2023-2976
+    events:
+      - timestamp: 2023-10-05T13:50:00Z
+        type: false-positive-determination
+        data:
+          type: component-vulnerability-mismatch
+          note: CVE already fixed in guava version flagged by scanner (v32.0.0). See https://github.com/google/guava/releases/tag/v32.0.0 and https://github.com/keycloak/keycloak/pull/21544
```

Comment:
Help me understand this one a bit better, I don't think I quite follow. Did the scanner parse the version wrong or something? Why does the scanner think this is a vulnerability, and what's wrong about its logic?

Reply:
Sure, in this case the scanner is flagging a false positive. For example:

[/usr/share/java/keycloak-22.0.4/lib/lib/main/com.google.guava.guava-32.0.0-jre.jar
│       📦 com.google.guava.guava 32.0.0-jre (java-archive)
│           High CVE-2023-2976](url)

It's correctly picked up that the installed version of guava is 32.0.0, but it's wrongly flagging CVE-2023-2976, which is indeed already fixed in this release.

Perhaps could be worded better but wasn't sure how to describe and keep high level

Reply:
I'd like to understand what's happening here to make the scanner flag the vulnerability. Could it be the version suffix of `-jre`? (Somehow the scanner thinks it's doing the right thing, so I just want to understand why 😄 )

Reply:
Happy to file a grype ticket, but if theres anything we can do on our side as well lets sync up tomorrow

Reply:
Hey @luhring, heres the grype ticket:
- https://github.com/anchore/grype/issues/1543

Let me know if you still want to take a look further but there should be plenty of details in there for the grype team.